packages/filesystem/src/browser/filesystem-frontend-contribution.ts

```diff
@@ -77,6 +77,9 @@ export class FileSystemFrontendContribution implements FrontendApplicationContri
     @inject(FileService)
     protected readonly fileService: FileService;
 
+    protected editorFileDidChangeEmitter = new Emitter<{ editor: NavigatableWidget, type: FileChangeType }>();
+    readonly onDidChangeEditorFile = this.editorFileDidChangeEmitter.event;
```

Comment:
minor - generally the emitter and event have similar names:

```suggestion
    protected onDidChangeEditorFileEmitter = new Emitter<{ editor: NavigatableWidget, type: FileChangeType }>();
    readonly onDidChangeEditorFile = this.onDidChangeEditorFileEmitter.event;
```

Reply:
👍